### DISCUSSION TITLE
可否提供6.48的jp的rawexd

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#382] [General]
抱歉打擾了 不知是否可以提供6.48 的 JP 的rawexd 謝謝大大提供

**Comment by GpointChen** [comment]
[rawexd-cut-ja648.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/12577828/rawexd-cut-ja648.zip)
只保留文字內容的rawexd。

[rawexd-ja-648.z01.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/12577866/rawexd-ja-648.z01.zip)
[rawexd-ja-648.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/12577871/rawexd-ja-648.zip)
包含非文字內容的rawexd，請將兩個檔案放在一起，`rawexd-ja-648.z01.zip`改名為`rawexd-ja-648.z01`後解壓縮。

你也可以使用 https://github.com/GpointChen/SaintCoinach/tree/hexcode 自行輸出CSV。
